site/content/integrate/apps/deployment-aws/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Deployment (AWS Serverless)"
+heading: "Deploying AWS Serverless Apps"
+description: "TODO"
+weight: 110
+---
+
+Serverless hosting allows for easy app installation from the App Marketplace by a System Admin and uses AWS Lambda serverless technology instead of relying on a physical server. Developers who create apps using a serverless development approach can easily deploy apps securely, efficiently, and at scale in the Mattermost Cloud.
```

Comment:
That doesn't seem right. The Marketplace has nothing to do with the deployment mechanism. Also, this section is not limited to MM Cloud.

Reply:
Fixed in 24377f0e